extensions/wikia/Email/i18n/Founder.i18n.php

```diff
@@ -22,6 +22,41 @@
+	'emailext-founder-add-pages-header' => 'Add Pages',
+	'emailext-founder-add-pages-details' => 'Go ahead, add everything you know about your favorite fandom. Share your knowledge and recruit others to help!',
+	'emailext-founder-add-photos-header' => 'Photos & Videos',
+	'emailext-founder-add-photos-details' => 'Load up your wikia with awesome media! Add photos and videos to your community and keep them coming back for more.',
+	'emailext-founder-customize-header' => 'Customize It',
+	'emailext-founder-customize-details' => 'What’s the latest news on your wikia? Feature the best content on your homepage to make a great first impression!',
+	'emailext-founder-share-header' => 'Share It',
+	'emailext-founder-share-details' => 'Love your wikia? Stand on a mountain and shout it to the world! Tag @Wikia on your favorite social media network.',
+	'emailext-founder-visit-community' => "Now you're on your way! But, don't worry, we won't leave you high and dry. We're here to help you make $1 successful every step of the way. Visit [http://community.wikia.com Community Central] to ask questions and get advice!",
+	'emailext-founder-happy-wikia-building' => "Happy wikia building!",
+	'emailext-founder-3-days-subject' => "How’s it going on $1? We’re here to help!",
+	'emailext-founder-3-days-summary' => 'Can you believe it’s been three days since you started $1?',
+	'emailext-founder-3-days-extended-summary' => "We're checking in to share a few more tips on building your wikia community and make sure you’re having a great fan experience.",
+	'emailext-founder-3-days-add-videos-header' => "Add Videos",
+	'emailext-founder-3-days-add-videos-details' => "Make your page awesome and interesting by adding your favorite fandom videos. And who doesn’t love a good video?",
+	'emailext-founder-3-days-update-theme-header' => "Update Your Theme",
+	'emailext-founder-3-days-update-theme-details' => "Themes, colors, wordmarks, oh my! Use the Wikia Theme Designer to create a design that matches your fandom perfectly.",
+	'emailext-founder-3-days-wam-header' => "Get Inspired",
+	'emailext-founder-3-days-wam-details' => "Draw inspiration from some of our [$1 favorite communities]. These represent the best and most popular wikia fan sites!",
+	'emailext-founder-3-days-need-help' => "Need help figuring out how something works? We're always here for you! Ask us for help and advice at [http://community.wikia.com Community Central].",
+	'emailext-founder-3-days-great-work' => "Keep up the great work! ",
+	'emailext-founder-10-days-subject' => "Happy 10-day anniversary on $1!",
+	'emailext-founder-10-days-summary' => "It’s been 10 days since you started $1.",
+	'emailext-founder-10-days-extended-summary' => "We must say, your wikia are looking great.  Now it’s time to spread the word!",
```

Comment:
"your wikia are looking great" seems wrong. Can you double-check the copy for this message?